### PR TITLE
testing: add api note about new lines

### PR DESCRIPTION
**Pull request by connor4312**
See #174771

### CODE REVIEW

**Review by Tyriar — APPROVED** [review]

File: src/vscode-dts/vscode.d.ts [hunk 1]
```diff
@@ -16215,7 +16215,8 @@ declare module 'vscode' {
 		/**
 		 * Appends raw output from the test runner. On the user's request, the
 		 * output will be displayed in a terminal. ANSI escape sequences,
-		 * such as colors and text styles, are supported.
+		 * such as colors and text styles, are supported. New lines must be given
+		 * as CRLF (`\r\n`) rather than LF (`\n`).
```
Comment (120210313069) [inline src/vscode-dts/vscode.d.ts:16219]: `\r\n`

Comment (120210313069) [reply src/vscode-dts/vscode.d.ts:16219]: __

Comment (120210313069) [reply src/vscode-dts/vscode.d.ts:16219]: (`\r\n`)

**Review by 120210313069 — COMMENTED** [review]
( \r\n)

**Pull request merged**